ffi/ecma402/build.rs

```diff
@@ -2,24 +2,26 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+use icu_datagen::*;
```

Comment:
Nit: This is not super pretty because the `icu_datagen` namespace has some things inside that aren't intended to be glob-imported, like the `keys` and `keys_for_bin` functions, etc.

Reply:
I made a prelude.